src/nxt_cgroup.c

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) Andrew Clayton
+ * Copyright (C) F5, Inc.
+ */
+
+#include <nxt_main.h>
+#include <nxt_process.h>
+#include <nxt_fs.h>
+
+#include <stdio.h>
+#include <string.h>
+#include <limits.h>
+
+
+static nxt_int_t nxt_make_cgpaths(const nxt_process_t *process, char *cgpath,
+    size_t cgpath_len, char *cgroot, size_t cgroot_len);
+static nxt_int_t nxt_cgroup_create(const nxt_task_t *task,
+    nxt_process_t *process);
+nxt_int_t nxt_cgroup_proc_add(const nxt_task_t *task, nxt_process_t *process);
```

Comment:
`extern` function prototypes duplicated from header.  I guess a glitch?

Reply:
Heh probably. I'm not used to putting function prototypes in. I always order my functions so prototypes aren't needed (well there is the odd exception). In fact this is the first project (that comes to mind) I've worked on that does this.

Reply:
Hmm, indeed, there's a mix of styles in the code, where some files try to order static functions so that there are less prototypes.  But personally, I prefer having them all, even if unnecessary.